plugins/notify/Notification.vala

```diff
@@ -160,20 +160,26 @@ namespace Gala.Plugins.Notify
 		public void open () {
 			var entry = new TransitionGroup ();
 			entry.remove_on_complete = true;
-			entry.duration = 400;
 
 			var opacity_transition = new PropertyTransition ("opacity");
 			opacity_transition.set_from_value (0);
 			opacity_transition.set_to_value (255);
 
-			var flip_transition = new KeyframeTransition ("rotation-angle-x");
-			flip_transition.set_from_value (90.0);
-			flip_transition.set_to_value (0.0);
-			flip_transition.set_key_frames ({ 0.6 });
-			flip_transition.set_values ({ -10.0 });
-
+			var slide_in_transition = new PropertyTransition ("anchor-x");
```

Comment:
Create this object only when the `} else {` branch is getting executed, that way, you don't create this unnecessarily.